main-command/src/main/scala/sbt/Watched.scala

```diff
@@ -50,18 +62,46 @@ trait Watched {
 }
 
 object Watched {
-  val defaultWatchingMessage: WatchState => String = ws =>
-    s"${ws.count}. Waiting for source changes... (press enter to interrupt)"
+  sealed trait Action
```

Comment:
These probably should be generated using Contraband.

Reply:
I use Contraband for enums as well - https://github.com/sbt/sbt/blob/develop/main/src/main/contraband/main.contra
But I just noticed that Action is not completely sealed because of Custom, so that might not be a good fit.

Reply:
Aah, makes sense. I am not opposed to contraband. The `Custom` action is sort of speculative in the event that users might want to use ~ to potentially trigger any arbitrary state transformation. My guess is that very, very few users would ever actually do this (even if I think it's a cool capability).